message_processor.go

```diff
@@ -27,11 +31,53 @@ func (mg messageProcessor) process(lf *logPair) {
 	consumeLog(lf)
 }
 
+type tmpBuffer struct {
+	tBuf                      bytes.Buffer
+	bufferTimer               time.Time
+	bufferReset               bool
+}
+
+func (b *tmpBuffer) appendToTempBuffer(l *logdriver.LogEntry) bool {
```

Comment:
I would suggest not swallowing the error, return it. So we need change `bool` to `error`

Reply:
@chenziliang 
Is it ok if I take care of this refactor as part of a different ticket? With moving the temp buffer to a different file or may be as an interface.